src/imgui/imgui_raii_wrapper.h

```diff
@@ -0,0 +1,162 @@
+#ifndef MYPROJECT_IMGUI_RAII_WRAPPER_H
+#define MYPROJECT_IMGUI_RAII_WRAPPER_H
```

Comment:
Why do you use this?

Reply:
The `#ifndef` guard will ensure that the contents of the header file `src/imgui/imgui_raii_wrapper.h` is added only once and be ignored If the header is included again.

Reply:
https://www.educative.io/edpresso/what-are-header-guards

Reply:
I wouldn't pay too much attention to this. It's a C/C++ quirk that hopefully won't be so necessary once modules are working in C++20. Any decent IDE will generate this for you for every header file, so just leave it there.